### DISCUSSION TITLE
Zillion unstake, how to get zil back on zilpay wallet after 100% progress

### DISCUSSION THREAD

**Discussion by donnieanton** [Zilliqa/staking-contract#221] [Q&A]
I am sorry to bother you here but I don't know what else to do or who to turn to I am using stake.zilliqa.com and I went through the withdrawal process of unstaking its at 100% the problem is when I go to withdraw my zil funds I'd like them to go back to my zilpay wallet it shows that I can now withdrawal zil but when I click the button to complete withdrawal it, zipay wallet, only allows me to send 0 zil back to the stakepool it doesn't allow for a me to withdraw from the stake pool back to my to my zilpay wallet and I'm just wondering if somebody might be able to help me.   I staked on March 10 and changed my mind to unstake and start withdraw process on March 11.  On March 26, 11136.642 zil became available for me to withdraw and shows in my account but I cannot withdraw back to zilpay wallet.  FYI I may have pressed the withdraw button twice on March 11 to withdraw 0 zil after I initiated the first withdraw request.

I think there is a problem with this withdrawal transaction: (i think for accepted it reads false, so did it work)
https://viewblock.io/zilliqa/address/zil1phf5ae0vzcva0t62q709nf2jcss5qcvau5ffl9?txsType=contract
the WithdrawStakeAmt amount shows 0 zil and I believe it should be 11136.642 zil

My initial stake delegation is here:
https://viewblock.io/zilliqa/tx/0x461888b693e70515428b7be0749a43b227a42ce2c92647ad0d1fa03517893a8c

So my question is that I believe it is associated with my zilpay wallet address zil1phf5ae0vzcva0t62q709nf2jcss5qcvau5ffl9 but in limbo right now not knowing where or what the system should do with it.  I cannot re-stake it or withdraw it.  I think it is just stuck on the Seed Node Staking Implementation Address: zil1k7qwsz2m3w595u29je0dvv4nka62c5wwrp8r8p

**Comment by donnieanton** [comment]
I searched around and downloaded moonlet app.  Moonlet showed my balance included the amount and allowed me to see the withdraw amount, so I did it.  The address showed from me to what looked like a staking address but by then the transaction was on its way and I just assumed staking withdraws don't show the to address as my own wallet on zilpay or moonlet.  So I got the funds out and where I want them but it is very confusing and fud out to the max.  I have a degree in computer science and it was hard for me,  you should fix that withdrawal interface.

**Comment by bb111189** [comment]
You have successfully withdraw it at https://viewblock.io/zilliqa/tx/0x20ad0d3966017dd294dd89f38c3f8edd86a3753046e2a36c31742a8a7e773915

**Reply by donnieanton** [reply]
Yes I replied last night when I was successful.  Zilpay wallet should be fixed to show the withdrawal amount instead of 0 on withdrawal transactions.  Once I restored to Moonlet app, moonlet showed me the 11k zil amount instead of just 0.   Both wallets showed the transfer from my address to a staking address which is also confusing. I was expecting to see from staking wallet to my address on transfer.  Maybe an explanation of the withdrawal process would help, I found others with same issues but Noone had answers in forums.  Do it yourself using zilpay wallet and zillion stake or zillet.io and you will see what I mean.
